stactools_pipelines/cdk/app.py

```diff
@@ -1,16 +1,17 @@
-import os
-
 import aws_cdk as cdk
 import yaml
 
 from stactools_pipelines.cdk.lambda_stack import LambdaStack
 from stactools_pipelines.models.pipeline import Pipeline
+from stactools_pipelines.models.deployment import Deployment
+
+deployment = Deployment()
```

Comment:
@emileten As mentioned in the initial comment I'd ideally like to see these `deployment` logic changes in a separate PR so we don't have a lot of unrelated commits intermingled.

Reply:
Agreed, I'll keep that in mind for the next times. If that's okay, for this time I'll keep this change here since it's not large.